frontend/src/model/Filters.ts

```diff
@@ -7,14 +7,24 @@ export interface CacheTypes {
     vainRatkaistutMultit?: boolean;
 }
 
+export interface CacheSize {
+    mikro?: boolean;
+    pieni?: boolean;
+    normaali?: boolean;
+    suuri?: boolean;
+    virtuaali?: boolean;
+    muu?: boolean;
+    tuntematon?: boolean;
+}
+
 export const initFilters: Filters = {};
 
 export interface Filters{
     limit?: number
     customRule?: string
     maxDistance?: number
     cacheTypes?: CacheTypes
-    size?: Array<string>
+    size?: CacheSize
```

Comment:
Here is a question... Size was originally type Array<string>. Is this necessary? I have changed it in my solution similar with preceding one, cacheTypes? is type of CacheTypes. See above export interface Cachetypes and CacheSize. Is this solution okay?

Reply:
Yup, looks good! The interfaces turned out to be much easier to work with than arrays, so let's use those for the filters whenever it makes sense!